ValveResourceFormat/Resource/ResourceTypes/Sound.cs

```diff
@@ -144,13 +169,19 @@ public override void Read(BinaryReader reader, Resource resource)
             SampleCount = reader.ReadUInt32();
             Duration = reader.ReadSingle();
 
-            // Skipping over m_Sentence (CSentence_t) and m_pHeader
-            reader.BaseStream.Position += 12;
+            var sentenceOffset = reader.ReadInt64();
```

Comment:
Is it actually long?

Reply:
Yeah probably isn't. Will revert it now. Reads fine though on all vsnds (next int is always 0x00).

Reply:
There are a bunch of zeroes, like all the others where we check for zero and throw if it's not (to find any).